docker/prepare_proxy_test.sh

```diff
@@ -0,0 +1,238 @@
+#!/bin/sh
```

Comment:
I'd really prefer not have these long bash test scripts. They are close to unmaintainable...